addons/mail/static/src/new/web/messaging_menu/notification_item.xml

```diff
@@ -2,28 +2,30 @@
 <templates xml:space="preserve">
 
 <t t-name="mail.notification_item" owl="1">
-    <div class="o-mail-notification-item d-flex cursor-pointer align-items-center p-1" t-att-class="{ 'border-bottom': !props.isLast, 'o-muted': !props.hasMarkAsReadButton }" t-on-click="onClick">
-        <div class="position-relative o-mail-background-inherit m-1 flex-shrink-0" style="width:40px;height:40px;">
-            <img class="w-100 h-100 rounded-circle" alt="Notification Item Image" t-att-src="props.iconSrc"/>
-            <t t-slot="icon"/>
-        </div>
-        <div class="d-flex flex-column flex-grow-1 align-self-start m-2 overflow-auto">
-            <div class="d-flex">
-                <span class="o-mail-notification-item-name fw-bold text-truncate" t-esc="props.displayName"/>
-                <span class="o-mail-notification-item-counter mx-1 fw-bold" t-if="props.count > 1"> (<t t-esc="props.count"/>)</span>
-                <span class="flex-grow-1"/>
-                <small t-if="props.datetime" class="o-mail-notification-item-date text-muted opacity-50 ms-2">
-                    <RelativeTime datetime="props.datetime"/>
-                </small>
-
+    <ActionSwiper onLeftSwipe = "hasLeftSwipe ? { action: () => this.props.onSwipeLeft(true), bgColor: 'bg-success', icon: 'fa-check-circle' } : undefined" onRightSwipe = "hasRightSwipe ? { action: () => this.props.onSwipeRight(true), bgColor: 'bg-danger', icon: 'fa-times-circle' } : undefined">
```

Comment:
seeing all previous issues and the way to fix them, it looks like the NotificationItem props/API in regard to swiper is really not good